packages/@dcl/sdk/cli/commands/init/repos.ts

```diff
@@ -1,5 +1,17 @@
-export const REPOS = {
-  'scene-template': 'https://github.com/decentraland/sdk7-scene-template/archive/refs/heads/main.zip'
+type Scene = 'scene-template'
+
+type Repos = {
+  [key in Scene]: {
+    url: string
+    contentFolders: string[]
+  }
+}
+
+const REPOS: Repos = {
+  'scene-template': {
+    url: 'https://github.com/decentraland/sdk7-scene-template/archive/refs/heads/main.zip',
+    contentFolders: ['sdk7-scene-template-main']
```

Comment:
instead of having `type Repo = { url: string, contentFolders: string[] }` why don't we just have each repo be a string like `decentraland/sdk7-scene-template` and then generate the url and content folder from that?

ie:

```ts
function getRepoUrl(repo: string) {
  return `https://github.com/${repo}/archive/refs/heads/main.zip`
}

function getRepoContentFolder(repo: string) {
  const [_organization, repository] = repo.split('/')
  return `${repository}-main`
}
```

That way can we keep it, also there shouldn't be multiple content folders since github wraps the content of the repo in a single directory

Reply:
didn't want to get attached to github URL's in case we add other templates from some cdn, etc